### PR TITLE
update sdl.js to use scalar datatypes in input types

### PR DESCRIPTION
**Pull request by chris-hailstorm**
Resolves the problem cited in [#210](https://github.com/redwoodjs/redwood/issues/210)

Per See https://graphql.org/graphql-js/mutations-and-input-types: "Input types can't have fields that are other objects, only basic scalar types, list types, and other input types".

In `generate scaffold`, identify custom types referenced by the model. Load model definitions for those types. Pass definitions to the `modelFieldToSDL` function. Identify the scalar data type used as the `@id` field in the referenced type and use that as the field type for the Input.

This changes the SDL for ___Input types only.  No change to base type, query or mutations.

### CODE REVIEW

File: packages/cli/src/commands/generate/commands/sdl.js [hunk 1]
```diff
@@ -39,17 +42,29 @@ const idType = (model) => {
 }
 
 const sdlFromSchemaModel = async (name) => {
+  console.log('FRED')
```
Comment (cannikin) [inline packages/cli/src/commands/generate/commands/sdl.js]: FRED 😄

**Comment by chris-hailstorm** [comment]
I pushed the wrong version!

Will de-Fred and resubmit.

**Comment by chris-hailstorm** [comment]
FRED resolved in fe313c51

**Comment by cannikin** [comment]
Love it, thanks so much! Any chance you'd like to join us in our Slack channel? You're basically a core contributor at this point!

**Comment by chris-hailstorm** [comment]
I’m up for that

**Comment by cannikin** [comment]
Great! Can you shoot me your email address? rob@prestonwernerventures.com

**Pull request merged**